nxdrive/data/qml/AccountsTab.qml

```diff
@@ -73,8 +73,10 @@ Rectangle {
                 columnSpacing: 50
                 rowSpacing: 20
                 property bool authenticated: !api.has_invalid_credentials(uid)
+                property bool sync_feature_enabled: feat_synchronization.enabled
```

Comment:
You could have done:
```suggestion
                property bool sync_enabled: feat_synchronization.enabled
```
And then no need to change more stuff in the file. But this is optional.